### DISCUSSION TITLE
When using a configuration, is there a clean way to initialize default values?

### DISCUSSION THREAD

**Discussion by ArchieCoder** [unoplatform/uno#19067] [Q&A]
I have a configuration that I want some booleans to True. I haven't found a clean way yet.

The help page does not mention it.
https://platform.uno/docs/articles/external/uno.extensions/doc/Learn/Tutorials/Configuration/HowTo-Configuration.html

Thanks

**Comment by nickrandolph** [comment]
You can set default values the same way as you would if you wanted to ship configuration with your application (ie using appsettings.json). This can be used for both readonly value (ie what we would refer to as Configuration) but also values the user might modify (ie often referred to as Settings)

**Reply by ArchieCoder** [reply]
How does it work if I have multiple configurations? I see that the default file in the Uno project is appsettings.json.

![image](https://github.com/user-attachments/assets/4bc6658e-5540-4739-afb5-5855808f7359)

**Reply by ArchieCoder** [reply]
Update: there is some magic with the appsettings.json file!

I pasted my other configuration and it is shown underneath the appsettings file. Unfortunately, when reading the configuration values the first time, everything is intialized at 0, false.

![image](https://github.com/user-attachments/assets/9d7cf09d-b9a6-415c-8535-d3f1a813837b)
